library/src/main/java/com/flipkart/layoutengine/view/DataProteusView.java

```diff
@@ -269,18 +252,10 @@ public void set(String dataPath, JsonElement newValue, int childIndex) {
             return;
         }
 
-        String aliasedDataPath = DataContext.getAliasedDataPath(dataPath,
-                parserContext.getDataContext().getReverseScopeMap(), true);
+        String aliasedDataPath = DataContext.getAliasedDataPath(dataPath, parserContext.getDataContext().getReverseScopeMap(), true);
 
-        JsonElement parent = null;
-        try {
-            parent = Utils.getElementFromData(aliasedDataPath.substring(0, aliasedDataPath.lastIndexOf(".")),
-                    parserContext.getDataContext().getDataProvider(), childIndex);
-        } catch (JsonNullException | NoSuchDataPathException | InvalidDataPathException e) {
-            if (ProteusConstants.isLoggingEnabled()) {
-                logger.error("#set() " + e.getMessage());
-            }
-        }
+        Result result = Utils.getElementFromData(aliasedDataPath.substring(0, aliasedDataPath.lastIndexOf(".")), parserContext.getDataContext().getDataProvider(), childIndex);
```

Comment:
Can you please make "." as final static string? This will get called so many times so it is better it gets initialized only once.